packages/m/mariadb-connector-c/xmake.lua

```diff
@@ -65,6 +65,16 @@ package("mariadb-connector-c")
             end
         end
         import("package.tools.cmake").install(package, configs)
+
+        if package:is_plat("windows") then
+            for _, lib in pairs(os.files(path.join(package:installdir("lib"), "mariadb", "*.lib"))) do
+                os.ln(lib, path.join(package:installdir("lib"), path.filename(lib)))
```

Comment:
please do not use os.ln on windows. Using soft link on windows requires administrator privileges and usually fails.